databricks/koalas/groupby.py

```diff
@@ -2652,7 +2667,7 @@ def _apply_series_op(self, op, should_resolve: bool = False, numeric_only: bool
             return kser
 
     @property
-    def _kdf(self) -> DataFrame:
+    def _kdf(self) -> DataFrame:  # type: ignore
```

Comment:
Disable type check, otherwise, `databricks/koalas/groupby.py:2672: error: Signature of "_kdf" incompatible with supertype "GroupBy"`.

Reply:
Then how about use `# type: ignore[override]` to make it clearer that it is an incompatible override error ?

Reply:
Great idea! Added it.